terraform/modules/google-cloud/apps/elixir/network.tf

```diff
@@ -70,7 +208,15 @@ resource "google_compute_backend_service" "default" {
   timeout_sec                     = 86400
   connection_draining_timeout_sec = 120
 
-  enable_cdn       = false
+  enable_cdn = var.application_cdn_enabled
```

Comment:
All our static assets are digested by Phoenix so we can use CDN to cache them.

Reply:
👍